addons/website/static/tests/tours/link_tools.js

```diff
@@ -225,8 +242,9 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     {
         content: "Change it back http",
         trigger: "#o_link_dialog_url_input",
+        extra_trigger: "div#oe_snippets:not(div.o_we_ui_loading)",
```

Comment:
Can you add a `TODO` on all those `o_we_ui_loading` extra loading? Those should not be here, it's surely due to a real race condition in our code
As discussed here https://github.com/odoo/odoo/pull/162031/commits/a4ed03b97e96a3b228b5a48d9c5ab4dc138950b7#r1572365856

Reply:
I was able to remove a majority of them